p2p/src/peers/mod.rs

```diff
@@ -490,6 +490,22 @@ pub fn split_socket_addresses(socket_addr: &SocketAddr) -> (Vec<u8>, Vec<u8>, Ve
     }
 }
 
+/// Returns the /8 IP range
+pub fn get_range_address(socket_addr: &SocketAddr) -> Vec<u8> {
```

Comment:
It would be nice to have a range argument here so that next time we want to change it it's just a one line change, but I guess it's better to do that in a separate PR?

```suggestion
pub fn get_range_address(socket_addr: &SocketAddr, range: u8) -> Vec<u8> {
```

Reply:
I agree, it could be better in a separate PR